src/Helpers/Services/actions.jsx

```diff
@@ -202,4 +202,65 @@ export const actionDownvoteComment = async (data) => {
       // showErrorToast(response.data.errors[0])
       showErrorToast("Unexpected error")
   }
-};
\ No newline at end of file
+};
+
+//actionLikeComment
+export const actionLikePost = async (data) => {
+  const response = await mainApiService("actionLikePost", data);
+  if (response.data &&(response.status === 201 || response.status === 200)) {
+    return response;
+    }else{
+      console.log("response.data.errors:", response.data.errors);
+      // showErrorToast(response.data.errors[0])
+      showErrorToast("Unexpected error")
+  }
+};
```

Comment:
can be error handled with try catch blocks

Reply:
Done that is  postslice.js